modules/@ergonode/workflow/src/components/Buttons/CreateWorkflowTransitionFab.vue

```diff
@@ -0,0 +1,40 @@
+/*
+ * Copyright © Ergonode Sp. z o.o. All rights reserved.
+ * See LICENSE for license details.
+ */
+<template>
+    <Fab
+        :size="regularSize"
+        :disabled="!isAllowedToCreate">
+        <template #icon="{ color }">
+            <IconAdd :fill-color="color" />
+        </template>
+    </Fab>
+</template>
+
+<script>
+import {
+    SIZE,
+} from '@Core/defaults/theme';
+import Fab from '@UI/components/Fab/Fab';
+import IconAdd from '@UI/components/Icons/Actions/IconAdd';
+import PRIVILEGES from '@Workflow/config/privileges';
+
+export default {
+    name: 'CreateWorkflowTransitionFab',
+    components: {
+        Fab,
+        IconAdd,
+    },
+    computed: {
+        isAllowedToCreate() {
+            return this.$hasAccess([
+                PRIVILEGES.WORKFLOW.create,
+            ]);
+        },
+        regularSize() {
+            return SIZE.REGULAR;
```

Comment:
Why is it computed? It is the same scenario like in icon file above, regularSize value will never be recalculated, what is more - vue will create setters / getters for computed properties even when they will never change, so they will get unecessary reactivity listeners.
edit: I see this pattern also in other files

Reply:
This convention was adopted a long time ago. changing it is quite time-consuming because we use it in many places.   We are aware of the problem with the computed property and will think about fixing it.